alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/EventListenerTask.java

```diff
@@ -136,6 +136,22 @@ public void run() {
                                         case "error":
                                             log.warn("Error instance status in deploymentID: {} and nodeID: {}", paasId, eNode);
                                             break;
+                                        case "submitting":
+                                        case "submitted":
+                                        case "executing":
+                                        case "executed":
+                                        case "running":
+                                        case "pending":
+                                        case "failed":
+                                        case "succeeded":
+                                        case "unknown":
+                                        case "No pods created":
+                                        case "completed":
+                                        case "completing":
+                                        case "signaling":
+                                        case "resizing":
+                                            // currently managed states that correspond to k8s and slurm job statuses
+                                            break;
                                         default:
                                             log.warn("Unknown instance status: " + eState);
```

Comment:
Why we don't just ignore unknown status as we no more respect Tosca node states ?

Reply:
Yes, this is better. I updated the PR descrition also.